docs/docsource/Operation-Architecture/Orbit_UI/Bucket_Management/Lifecycle_Management.rst

```diff
@@ -30,7 +30,7 @@ however, the following transition rules can only be defined using API calls.
    * Setting the Days field to zero
    * Setting a custom ID
 
-See :ref: 'bucket_lifecycle_operations'
+See "Bucket Lifecycle Operationss" in the *Reference Guide*.
```

Comment:
`Operations`

Reply:
Dadblameit! I fixed that once already. 

I fixed it again.